hoot-js/src/main/cpp/hoot/js/schema/JavaScriptSchemaTranslator.cpp

```diff
@@ -285,10 +285,10 @@ void JavaScriptSchemaTranslator::_init()
   {
     _toOsmFunctionName = "translateAttributes";
   }
-  else
-  {
-    throw HootException("A 'translateToOsm' function must be defined.");
-  }
+//  else
+//  {
+//    throw HootException("A 'translateToOsm' function must be defined.");
```

Comment:
Temporarily removed this so that I can split toOgr and toOsm into different files. 
With this code enabled, an Export script (`translateToOgr`) must have either a `translateAttributes` or `translateToOSM` function or it will fail.
An Import script is not required to have an Export function.